Remove some unusued code

rwwagner90 opened the pull request:
Just doing some initial cleanup before I work on tackling pulling in more of the data for the builds from the json and automating things.

locks commented:
@rwwagner90 I also did a thing https://github.com/ember-learn/builds/pull/78/files 😅. What do you think about landing mine and then rebasing yours on that work?

rwwagner90 commented:
@locks ah man, I checked to make sure there were no PRs before I started. I guess you beat me too it 😛 . We can do that.

locks commented on package.json:
```diff
@@ -45,7 +46,7 @@
     "ember-moment": "^7.5.0",
     "ember-resolver": "^4.4.0",
-    "ember-source": "https://s3.amazonaws.com/builds.emberjs.com/canary/shas/6f92209039f4f9a2c08c5642a02a4e598f3702cd.tgz",
+    "ember-source": "~3.1.0-beta.1",
```
I'm confused by this, shouldn't my PR have this change? 🤔

rwwagner90 replied:
@locks I think yours was still on canary.

locks commented on package.json:
```diff
@@ -37,6 +37,7 @@
     "ember-cli-mirage": "^0.4.0",
     "ember-cli-moment-shim": "^3.4.0",
     "ember-cli-qunit": "^4.1.1",
+    "ember-cli-sass": "^7.1.5",
```
could we avoid sass? node-sass gives me twitches

rwwagner90 commented:
@locks in the spirit of getting things done and shipping code, I removed the sass, styles, and link-to active stuff that I changed, and just kept the remaining few places I removed some dead code.

locks commented:
:shipit:
